Description: schleuder config: Use default Debian keyserver
 Debian stretch (and all future versions) ship(s) with sensible
 keyserver defaults, therefore, use these.
 
Author: Georg Faerber <georg@riseup.net>
Last-Update: 2017-10-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/schleuder.yml
+++ b/etc/schleuder.yml
@@ -18,9 +18,11 @@
 #keyserver: hkp://jirk5u4osbsr34t5.onion
 # If you have an OS-wide defined keyserver, specify a blank value to have that
 # one used:
-#keyserver: 
-# The default works for all supported versions of gnupg:
-keyserver: pool.sks-keyservers.net
+#keyserver:
+
+# Debian stretch (and all future versions) ship(s) with sensible
+# keyserver defaults, therefore, use these:
+keyserver: 
 
 # Who is maintaining the overall schleuder installation and should be
 # notified about severe problems with lists.
